docs/articles/2023/20230808-r_api_release.md

```diff
@@ -0,0 +1,111 @@
+
+* [Quick start guide.](../../cellxgene_census_docsite_quick_start.md)
+* [R reference docs and tutorials.](https://chanzuckerberg.github.io/cellxgene-census/r/index.html)
+* Querying and slicing data tutorial << INSERT LINK >>.
+
+## Census R package is made possible by `tiledbsoma`
+
+The `cellxgene.census` package relies on [TileDB-SOMA](https://github.com/single-cell-data/TileDB-SOMA) R's package `tiledbsoma` for all of its data access capabilities as shown in the next section. 
+
+CZI and TileDB have worked closely on the development of `tiledbsoma` and recently upgraded  it from beta to its first stable version. << INSERT link their release notes >>.
+
+## Efficient access to single-cell data for >30M cells from R
+
+Census hosts ever-growing [data releases](../../cellxgene_census_docsite_data_release_info.md) from CZ CELLxGENE Discover, representing the largest aggregation of standardized single-cell data.
+
+Census data are accompanied by cell and gene metadata that have been standardized on ontologies across all datasets hosted in CZ CELLxGENE Discover. For example all cell types and tissues have been mapped to a value of the CL and UBERON ontologies, respectively. You can find more about the data in the [Census data and schema](../../cellxgene_census_docsite_schema.md) page.
+
+With the R package `cellxgene.census` researchers can have access to all of these data and metadata directly from an R session with following capabilities:
+
+
+### Easy-to-use handles to the cloud-hosted Census data
+
+From R users can get a handle to the data by opening the census.
+
+```r
+library("cellxgene.census")
+
+census <- open_soma()
+
+# Your work!
+
+census$close()
+``` 
+
+### Querying and reading single-cell metadata from Census
+
+Following our [Census data and schema](../../cellxgene_census_docsite_schema.md), users can navigate and query Census data and metadata by using any combination of gene and cell filters.
+
+For example, reading a slice of the human cell metadata for more than 300K cells with Microglial cells or Neurons from female donors:
+
+```r
+library("cellxgene.census")
+
+census <- open_soma()
+
+# Open obs SOMADataFrame
+cell_metadata = census$get("census_data")$get("homo_sapiens")$get("obs")
+
+# Read as an iterator of Arrow Tables
+cell_metadata = cell_metadata$read(
+   value_filter = "sex == 'female' & cell_type %in% c('microglial cell', 'neuron')",
+   column_names = c("assay", "cell_type", "sex", "tissue", "tissue_general", "suspension_type", "disease")
+)
+
+# Concatenate iterations
+cell_metadata <- cell_metadata$concat()
+
+# Convert to R tibble (dataframe)
+cell_metadata = as.data.frame(cell_metadata)
+
+census$close()
+```
+
+### Exporting Census slices to `Seurat` and `SingleCellExperiment`
+
+Similarly querying both the single-cell data along metadata can be easily exported to  `Seurat` or `SingleCellExperiment` object for downstream analysis:
```

Comment:
Does this suggested change maintain the intended meaning?
```suggestion
Similarly, querying both the single-cell data along with its metadata, it can be easily exported to  `Seurat` or `SingleCellExperiment` object for downstream analysis:
```